src/Riverline/MultiPartParser/Part.php

```diff
@@ -46,7 +46,7 @@ public function __construct($content)
     protected function parseContentString($content)
     {
         // Split headers and body
-        $splits = preg_split('/(\r?\n){2}/', $content, 2);
```

Comment:
I agreed and it was my first attempt to solve this problem ( see b674ce8171fd681455ace2129ba2b3b4f3316720 )
Unfortunaly, I had lot of problems with inconsistent end of line ( mixing \r\n and \n in the same document ).
As I said, we use this lib to parse multipart emails and it seems a lot of email clients and servers doesn't respect RFC ...